app/views/templates/ChangeSuccessView.scala.html

```diff
@@ -31,12 +31,36 @@ <h2 class="heading-medium">@messages("common.whatHappensNext")</h2>
 
 @if(user.isAgent) {
 
-  @model.agentEmail match {
-    case Some(email) => {<p>@messages("changeSuccess.helpOne.digital.agent", email)</p>}
-    case _ => {<p>@messages("changeSuccess.helpOne.paper.agent")</p>}
-  }
+  @if(appConfig.features.bulkPaperOffEnabled()) {
+
+    @defining(
+        model.businessName.getOrElse(messages("changeSuccess.agent.client.name"))
+    ) { clientName =>
+
+      @model.agentEmail.map { agentEmail =>
+        <p>
+          @messages("changeSuccess.agent.email.partOne")
+          <strong class="bold">@agentEmail</strong>
+          @messages("changeSuccess.agent.email.partTwo")
+        </p>
+      }
+
+      <p>
+        @messages("changeSuccess.agent.client.partOne")
+        <strong class="bold">@clientName</strong>
```

Comment:
Do we want this text to be bold even when it says "your client"? The `updateClient` helper keeps it in regular styling which I believe was the requirement at the time this was first implemented.

Reply:
Oops yeah good spot